music_assistant/controllers/streams.py

```diff
@@ -11,7 +12,7 @@
 
 from aiohttp import web
 
-from music_assistant.constants import FALLBACK_DURATION, SILENCE_FILE
+from music_assistant.constants import FALLBACK_DURATION, SILENCE_FILE, BASE_URL_OVERRIDE_ENVNAME
```

Comment:
```suggestion
from music_assistant.constants import (
    FALLBACK_DURATION, SILENCE_FILE, BASE_URL_OVERRIDE_ENVNAME
)
```

Reply:
otherwise the linter fails due to max line length

Reply:
Yea, I just noticed.. Didn't think about that :) thanks

Reply:
CI is still failing. Hold on I'll quickly run the linter against your code.